test/regressions/index.test.js

```diff
@@ -2,6 +2,8 @@ import * as fse from 'fs-extra';
 import * as path from 'path';
 import * as playwright from 'playwright';
 
+const ERROR_THRESHOLD = 2;
```

Comment:
I don't think those work. Either we allow 0 or any. Otherwise why 2? This doesn't guard against new errors since any change could remove one and add another.

Reply:
Ok fair enough. I got inspired by the `--max-warnings` option of eslint. I have used it in a medium-size dev team in the past, and it seemed to work (~5 people) to catch up with legacy. Considering we are going with `cross-env NODE_ENV=production`, I can remove this option. I will propose it a try in the material-ui-x repo, see how it goes.

Reply:
As usual, just because eslint has this, doesn't mean it's a good option. It makes sense for them for projects introducing eslint. But long term it should always be either 0 warnings or all. Otherwise the number of warnings doesn't tell you much. It would make sense to ignore certain warnings.